src/components/charts/SwipeChart.tsx

```diff
@@ -14,30 +7,23 @@ import {
   asString,
   asTuple
 } from 'cleaners'
+/* For our `handleGetPointerProps` handler: */
```

Comment:
Please remove this exception, and instead change the `props` variable to be called `opts` or something, inside `handleGetPointerProps`. The lint rule is picky about things called "props"